### ISSUE TITLE
Support Environment import and export

### ISSUE THREAD

**Issue by ChristophSchwenk** [usebruno/bruno#193]
Currently it't not possible to export environments and share them with the team. Also an import option of existing postman environments would be very helpful.

**Comment by helloanoop** [comment]
Hi @ChristophSchwenk 

In Bruno, When you export a collection as a file, the environments are also exported in the same file.

> Also an import option of existing postman environments would be very helpful.

Yes, We will support this soon.

**Comment by cardonator** [comment]
@helloanoop has any work been done yet on importing Postman environments?

**Comment by helloanoop** [comment]
Not yet. This should be rather straight forward parsing of a simple key value pair.

@therealrinku please see if you could pick this up when you have some time.
UX wise, I'd imagine a "Import button" just below the "+ Create" button in Bruno's Environment Config UI

**Comment by premeaswaran** [comment]
@helloanoop I am working on this currently. Will raise the PR asap unless someone else beats me up on it. Just need some help/inputs in testing out the changes in my local.

**Comment by helloanoop** [comment]
@premeaswaran Send a across the PR once it's ready. I will review it.

**Comment by nubunto** [comment]
was going to do it but I can wait!

**Comment by premeaswaran** [comment]
@helloanoop Please let me know if any changes required. Newbie here for js and react. So any kind of inputs welcomed!

**Comment by zachradtka** [comment]
I was just trying to export environment variables from bruno and currently I can export my collections but not my environments. I am on v1.11.0.

I might be missing something, but this issue is asking for support of import and export of environments and this was closed without support export.

**Comment by vazome** [comment]
@zachradtka yeah, I don't care much about postman migrations. I have a clean start, but it would be cool to share my environment to collaborate on.

Look into this one: https://github.com/usebruno/bruno/issues/2017

**Comment by iutale-lilly** [comment]
still not able to export env vars

**Comment by anusree-bruno** [comment]
> still not able to export env vars

As mentioned [here](https://github.com/usebruno/bruno/issues/193#issuecomment-1726356899), environment variables are exported along with Bruno collections.

Are you trying to export them individually? Could you please share a bit more about your use case? For importing and exporting Global Environments specifically, we are tracking that separately in [this issue](https://github.com/usebruno/bruno/issues/3529).

**Comment by iutale-lilly** [comment]
i've meant we are not able to export them to a postman collection.
we need to share the collections and envs with colleagues that do not use Bruno (for now), and it would be nice to be able to export also the envs

**Comment by minamz** [comment]
I'd really like to be able to clone an environment from one collection to another. We can do it by copying the environments folder in the explorer, but it would be much nicer / faster to have the functionality to do it from the GUI while moving between a lot of collections. Is there a way I missed?

**Comment by anusree-bruno** [comment]
Currently, there isn't a straight way to do this. We recommend setting these environments as global if they're shared across multiple collections.

That said, it does make sense to support importing and exporting environments at the collection level as well. I’ll reopen this issue to track that.

**Comment by minamz** [comment]
@anusree-bruno

 thank you for the consideration.

> We recommend setting these environments as global if they're shared across multiple collections.

this does not fit my use case, because
- if i did that, my global env vars would get really bloated
- I don't want to use those env vars for ALL collections, only specific ones 
- i still need to make adjustments to the the env vars on collection level

**Comment by paulverbeke** [comment]
> As mentioned [here](https://github.com/usebruno/bruno/issues/193#issuecomment-1726356899), environment variables are exported along with Bruno collections.
> 
> Are you trying to export them individually? Could you please share a bit more about your use case? For importing and exporting Global Environments specifically, we are tracking that separately in [this issue](https://github.com/usebruno/bruno/issues/3529).

First, thank you to all the Bruno team for this incredible software.

I'm also looking for a mean to export individual Bruno Environment to Postman Environment format. My colleagues aren't using Bruno and are always sharing Postman Collection/Environment files (through git) and expect that I send them (commit) Postman format as well when I edit some things.

Importing Postman collection/environment files in Bruno works well for me, but (still talking about Postman format) I can only export collections.

Yes when I export a Bruno collection to Postman format, I can see the variables from my Bruno Environment being translated to a "Postman collection-level variables". 
But :
- unfortunately it isn't what I'm looking for, I would like a dedicated Postman Environment file that my colleague can import into Postman
- apparently it export only the variables from the 1st Bruno Environment it found in the list of this Bruno Collection. Seems abitrary.
- the variables values aren't even included, despite them not being set as secrets

Additionally regarding the export of Bruno Collection to Postman format (maybe it's not the right place to include this):
- the Bruno Collection-level Auth of "Bearer Token" is completely lost
- the request-level Auth of "inherited" is lost also. Instead it's unfortunately set to "noauth"

Anyway, ideally I would have a button "Export in Postman format" in the "Export Environments" modal window right here
<img width="772" height="677" alt="Image" src="https://github.com/user-attachments/assets/d943ad8c-bf1e-4039-8fe3-c8abff4c503a" />

**Comment by helloanoop** [comment]
Closing this issue as we have implemented the core import and export around Bruno environments.
Exporting Bruno env to Postman Env is tracked in https://github.com/usebruno/bruno/issues/6197

Bruno now supports [Workspaces](https://docs.usebruno.com/get-started/bruno-basics/create-a-workspace), which are persisted on the filesystem just like collections. This makes it easy for teams to share environments and other workspace-level configuration across multiple collections, with everything stored in Git and collaboratively managed.
